devices/Lp3943/samples/Program.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Device.I2c;
+using System.Threading;
+using Iot.Device.Lp3943;
+using Iot.Device.Lp3943.Samples;
+
+I2cConnectionSettings settings = new(3, Lp3943.DefaultI2cAddress);
+using var device = new I2cDevice(settings);
+var resetPin = Utilities.GetPinNumber('D', 10);
+
+using var ledDriver = new Lp3943(device, resetPin);
+
+ledDriver.DimRegister(DimRegister.Dim0, 1, 25);
+ledDriver.DimRegister(DimRegister.Dim1, 5, 50);
+ledDriver.SetLed(
+	new[]
+	{
+		12,
+		13,
+		14,
+		15
+	},
+	LedState.Off
+);
+ledDriver.SetLed(
+	new[]
+	{
+		0,
+		1,
+		2,
+		3,
+		4,
+		5
+	},
+	LedState.On
+);
+ledDriver.SetLed(
+	new[]
+	{
+		6,
+		7,
+		8
+	},
+	LedState.Dim0
+);
+ledDriver.SetLed(9, LedState.Dim1);
+ledDriver.SetLed(10, LedState.Dim1);
+ledDriver.SetLed(11, LedState.Dim1);
+
```

Comment:
maybe add a Debug.WriteLine with the expected behavior or a simple comment in the code of what is expected here